### PR TITLE
Add lesson02

### PR DESCRIPTION
**Pull request by haraldF**

### CODE REVIEW

File: lesson02.md [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+3 !== 5           // true
+3 !== 3           // false
+1 !== "Cookies"   // true
+```
+
+---
+
+### Comparison operators
+
+We can use `===` and `!==` to test for the equality and inequality of any 2 values in javascript.
```
Comment (olof-nord) [inline lesson02.md]: ultra-nit: but one could aim for consistency and always write JavaScript. Its written like that in many places, but not everywhere. What do you think?

File: lesson02.md [hunk 2]
```diff
@@ -0,0 +1,440 @@
...
+---
+
+### Examples:
+
+```js
+let priceCoffee = 2;
+let priceCappuccino = 3;
+let customer = "John";
+let greeting = "Hello";
+let likesCarlo = true;
```
Comment (olof-nord) [inline lesson02.md:261]: What about using `likesAlan` instead? To keep to the Alan and Ada theme

File: lesson02.md [hunk 3]
```diff
@@ -0,0 +1,440 @@
...
+Create a new variable that contains how much we have to pay in total.
+<!-- .element: class="fragment" -->
+
+
+Ada and Alan volunteered to pay for the tickets. Use JavaScript to compute how much each has to pay.
+<!-- .element: class="fragment" -->
+
+---
+
+### Bonus
```
Comment (olof-nord) [inline lesson02.md:426]: I understand we wanted to name these Homework instead? As to clarify that they are tasks which are to be taken home.

File: lesson02.md (hunk 1 above)
Comment (haraldF) [reply lesson02.md]: Nice catch. Fixed.

File: lesson02.md (hunk 3 above)
Comment (haraldF) [reply lesson02.md:426]: It's a Tuesday's class, so we traditionally don't give homeworl

Comment (olof-nord) [reply lesson02.md:426]: got it, makes sense

**Pull request merged**